applications/ShapeOptimizationApplication/python_scripts/custom_variable_utilities.py

```diff
@@ -0,0 +1,41 @@
+# ==============================================================================
```

Comment:
If you think that those functions are general enough you can consider implementing them in C in the core, then also others could benefit

Reply:
Where would you put it then?